.github/workflows/onPullRequestCommit.yml

```diff
@@ -0,0 +1,91 @@
+name: Automatic Testing
```

Comment:
if you want the contest of this file to automaticTesting then you may as well name the filename the same

Reply:
renamed